app/src/main/res/layout/component_exceptions.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<LinearLayout
```

Comment:
Do we need the outer layout? Extra view hierarchy has a real cost.

Reply:
I think we do need this one? This is how the other components are created, passing a wrapper view that the component's UI views are added to

Reply:
Sounds fine to me. Just had to check.